common/src/main/java/com/wynntils/wynn/handleditems/annotators/gui/SkillPointAnnotator.java

```diff
@@ -26,18 +26,11 @@ public ItemAnnotation getAnnotation(ItemStack itemStack, String name) {
         String skillName = matcher.group(1);
         Skill skill = Skill.fromString(skillName);
 
-        int skillPoints = -1;
-        LinkedList<String> a = ItemUtils.getLore(itemStack);
-        for (String lore : ItemUtils.getLore(itemStack)) {
-            Matcher m = POINT_PATTERN.matcher(lore);
-            if (m.find()) {
-                String points = m.group(1);
-                skillPoints = Integer.parseInt(points);
-                break;
-            }
-        }
+        String loreLine = WynnUtils.normalizeBadString(ItemUtils.getLoreLine(itemStack, 3));
```

Comment:
hard coding the line number here breaks intelligence points, since that one has an extra line to indicate spell cost reduction:
![image](https://user-images.githubusercontent.com/3767283/211097948-e78822d7-639a-4a9e-ad56-725d3891e84c.png)
![image](https://user-images.githubusercontent.com/3767283/211097963-dfdb6dc7-acf9-4293-aebf-985e4aa3d82b.png)

Reply:
Fixed now